modin/core/dataframe/pandas/partitioning/axis_partition.py

```diff
@@ -65,7 +66,7 @@ def apply(
         """
         if num_splits is None:
             num_splits = len(self.list_of_blocks)
-
+        func_call = (func, args, kwargs)
```

Comment:
Elsewhere in Modin we are using `NamedTuple` for objects like this, e.g. [here](https://github.com/modin-project/modin/blob/9b94cb27bd7ea0b28ce6e6a163c056c3fa38d609/modin/config/pubsub.py#L20). I suggest switching to that.

Reply:
will switch to this after getting the dask virtual partition changed + passing tests.